### PR TITLE
Feature/adjustments

### PR DESCRIPTION
**Pull request by ghost**
## Summary

* Remove custom fonts
* Improve code

### CODE REVIEW

File: wbooks/src/app/screens/Library/index.tsx [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
 
   return (
-    <FlatList
-      showsVerticalScrollIndicator={false}
-      data={books}
-      keyExtractor={getKeyExtractor}
-      renderItem={renderItem}
-    />
+    <SafeAreaView style={styles.container}>
+      <View style={styles.subContainer}>
```
Comment (sfernandez11) [inline wbooks/src/app/screens/Library/index.tsx]: Move this style to `FlatList` style and remove unnecessary View

File: wbooks/src/app/screens/Library/index.tsx [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
-interface Props {
-  books: Book[];
-}
-
-function Library({ books }: Props) {
+function Library() {
   const getKeyExtractor = (item: Book) => `${item.id}`;
 
   const renderItem: ListRenderItem<Book> = ({ item }) => (
     <BookComponent id={item.id} imageUrl={item.imageUrl || ''} title={item.title} author={item.author} />
```
Comment (sfernandez11) [inline wbooks/src/app/screens/Library/index.tsx]: `imageUrl={item.imageUrl || ''}` => `imageUrl={item.imageUrl}` you must handle the null case in `BookComponent`

File: wbooks/src/utils/bookUtils.ts [hunk 3]
```diff
@@ -1,3 +1,3 @@
-const BOOK_PLACEHOLDER = require('@assets/book/img_book_placeholder.png');
+import bookPlaceholder from '@assets/book/img_book_placeholder.png';
 
-export const validateImageUrl = (url: string) => (url ? { uri: url } : BOOK_PLACEHOLDER);
+export const validateImageUrl = (url: string) => (url ? { uri: url } : bookPlaceholder);
```
Comment (sfernandez11) [inline wbooks/src/utils/bookUtils.ts]: string | null

**Review by sfernandez11 — CHANGES_REQUESTED** [review]

File: wbooks/src/app/screens/Library/index.tsx (hunk 2 above)
Comment (ghost) [reply wbooks/src/app/screens/Library/index.tsx]: Done!

File: wbooks/src/app/screens/Library/index.tsx (hunk 1 above)
Comment (ghost) [reply wbooks/src/app/screens/Library/index.tsx]: Done!

File: wbooks/src/utils/bookUtils.ts (hunk 3 above)
Comment (ghost) [reply wbooks/src/utils/bookUtils.ts]: Done!

**Review by sfernandez11 — APPROVED** [review]
🥇

**Pull request merged**